File: Branching statement/Question2/main.py
num1 = int(input("Enter first number: "))
num2 = int(input("Enter second number: "))

sum = num1 + num2
sub = num1 - num2
mul = num1 * num2
div = num1 / num2


def options(argument):
    if argument == 1:
        print(f"Sum of {num1} and {num2}:", sum)
    elif argument == 2:
        print(f"Subtraction of {num2} from {num1}:", sub)
    elif argument == 3:
        print(f"Multiplication of {num1} and {num2}:", mul)
    elif argument == 4:
        print(f"Division of {num1} and {num2}", div)
    elif argument == 5:
        print("Thank you for using the calculator")
        print("Exiting...")
        return
    else:
        print("Invalid input")
        return


print("Choose an option: \n")
print("1. Addition")
print("2. Subtraction")
print("3. Multiplication")
print("4. Division")
print("5. Exit")
choice = int(input())
options(choice)

File: Branching statement/Question2/main.cpp
#include <iostream>

int main()
{
    int num1, num2, sum, sub, mul, div, choice;

    std::cout << "Enter two numbers: ";
    std::cin >> num1 >> num2;

    std::cout << "1. Addition\n2. Subtraction\n3. Multiplication\n4. Division\n5. Exit\n";
    std::cout << "Enter your choice: ";
    std::cin >> choice;

    switch (choice)
    {
    case 1:
        sum = num1 + num2;
        std::cout << "Sum of " << num1 << " and " << num2 << " is " << sum << std::endl;
        break;
    case 2:
        sub = num1 - num2;
        std::cout << "Subtraction of " << num1 << " and " << num2 << " is " << sub << std::endl;
        break;
    case 3:
        mul = num1 * num2;
        std::cout << "Multiplication of " << num1 << " and " << num2 << " is " << mul << std::endl;
        break;
    case 4:
        div = num1 / num2;
        std::cout << "Division of " << num1 << " and " << num2 << " is " << div << std::endl;
        break;
    case 5:
        std::cout << "Exiting..." << std::endl;
        break;
    default:
        std::cout << "Invalid choice" << std::endl;
        break;
    }
    return 0;
}